kong/llm/proxy/handler.lua

```diff
@@ -477,4 +484,70 @@ function _M:access(conf)
 
 end
 
+
+
+function _M:init_worker_shared(plugin_name)
```

Comment:
let's just call it something like `build_http2_alpn_filter` to be more descriptive

Reply:
done